README.md

```diff
@@ -1,4 +1,6 @@
-# Popper
+<img src="docs/figures/popper_logo.png" width="200">
```

Comment:
This can be changed to the markdown syntax instead of mixing html and markdown.

Reply:
Yeah sure, the problem with the Markdown syntax is that it doesn't support a size parameter on GitHub. Take a look of how the logo looks like and if you're okay with the size I'll use that syntax. Or if you know a way to change the size using the Markdown syntax please let me know.

Reply:
This is simple. You can use a resized logo instead. You can just resize it and add a `small_logo.png` or something like that and then use it.

Reply:
I guess you're right. I made the changes :smile: